src/lib/phdi-building-blocks/phdi_building_blocks/linkage.py

```diff
@@ -1,7 +1,7 @@
 import hashlib
 
 
-def add_patient_identifier(salt_str: str, bundle: dict) -> None:
+def add_patient_identifier(bundle: dict, salt_str: str) -> dict:
```

Comment:
Why the switch here?

Reply:
I was trying to adopt the convention of always passing the bundle as the first argument for all of bundle transform building blocks. The change to returning a dict is so that we can pass by value.